### PR TITLE
aa updates

### PR DESCRIPTION
**Pull request by SahilAujla**
Live links in a test project:

- [Create Policy](https://alchemy-test.readme.io/reference/create-policy-4)
- [Get Policy](https://alchemy-test.readme.io/reference/get-policy-4)
- [Delete Policy](https://alchemy-test.readme.io/reference/delete-policy-4)
- [Replace Policy](https://alchemy-test.readme.io/reference/replace-policy-4)
- [Get All Policies](https://alchemy-test.readme.io/reference/get-all-policies-4)
- [Update Policy Status](https://alchemy-test.readme.io/reference/update-policy-status-4)
- [Get Policy Stats](https://alchemy-test.readme.io/reference/get-policy-stats-4)

### CODE REVIEW

File: account-abstraction/gas_manager_admin.yaml [hunk 1]
```diff
@@ -108,6 +128,10 @@ paths:
             schema:
               type: object
               properties:
+                policyName:
+                  type: string
+                  description: Name of the policy
+                  default: 'My Policy'
```
Comment (elanh) [inline account-abstraction/gas_manager_admin.yaml:134]: @SahilAujla  this is repeated a few times, should we make it a component?

**Review by elanh — APPROVED** [review]

File: account-abstraction/gas_manager_admin.yaml [hunk 2]
```diff
@@ -3,25 +3,28 @@ info:
   title: 🪄 Gas Manager Admin API
   version: '1.0'
 servers:
-  - url: https://manage.alchemy.com/
+  - url: https://manage.g.alchemy.com/
 paths:
-  '/api/gas-manager/policy':
+  '/api/gasManager/policy':
```
[the remark this answers is not in the document]
Comment (SahilAujla) [reply account-abstraction/gas_manager_admin.yaml:8]: yes

File: account-abstraction/gas_manager_admin.yaml [hunk 3]
```diff
@@ -3,25 +3,28 @@ info:
...
         content:
           application/json:
             schema:
               type: object
               properties:
-                app_id:
+                appId:
                   type: string
-                  description: String ID of the app
+                  description: ID of the app to which the newly created policy will belong
```
[the remark this answers is not in the document]
Comment (SahilAujla) [reply account-abstraction/gas_manager_admin.yaml:32]: Done

File: account-abstraction/gas_manager_admin.yaml (hunk 1 above)
Comment (SahilAujla) [reply account-abstraction/gas_manager_admin.yaml:134]: @elanh this is repeated only 2 times. I already tried to make a component out of it but was having some weird issues when I did and referenced it. 

Keeping in mind the current time limitations and other projects, I thought I would come back to this later to debug.

**Pull request merged**